### DISCUSSION TITLE
PyG latest version match with cuda 11.3

### DISCUSSION THREAD

**Discussion by soumbane** [pyg-team/pytorch_geometric#5909] [General]
Hi,

I have cuda 11.3 , Python 3.9, Pytorch 1.12.0 and PyG 2.1.0 installed. However, I could not find some packages in PyG utils such as add_random_edge and some others, which are only available in the latest version.
Could I install the latest version with pip install torch_geometric and use those utils? Will the latest version work with the above cuda and Pytorch configurations??

Please let me know.

Thanks.

**Comment by EdisonLeeeee** [comment]
PyG is not directly dependent on the CUDA and Python versions, so you can install [pyg-nightly](https://github.com/pyg-team/pytorch_geometric#nightly-and-master) to use these utils.

**Reply by soumbane** [reply]
Ok, got it. Thank you.
